### PR TITLE
updates for BZ1938270

### PR DESCRIPTION
**Pull request by aireilly**
Updates for https://bugzilla.redhat.com/show_bug.cgi?id=1938270

### CODE REVIEW

**Comment by aireilly** [comment]
https://deploy-preview-32540--osdocs.netlify.app/openshift-enterprise/latest/networking/multiple_networks/assigning-a-secondary-network-to-a-vrf.html

and 

https://deploy-preview-32540--osdocs.netlify.app/openshift-enterprise/latest/networking/hardware_networks/configuring-sriov-device.html#cnf-assigning-a-sriov-network-to-a-vrf_configuring-sriov-device

File: modules/cnf-assigning-a-sriov-network-to-a-vrf.adoc [hunk 1]
```diff
@@ -9,6 +9,11 @@ As a cluster administrator, you can assign an SR-IOV network interface to your V
 
 To do this, add the VRF configuration to the optional `metaPlugins` parameter of the `SriovNetwork` resource.
 
+[NOTE]
+====
+Applications which use VRFs need to bind to a specific device. The common usage is to use the `SO_BINDTODEVICE` option for a socket. `SO_BINDTODEVICE` binds the socket to a particular device as specified in the passed interface name, for example, `eth1`. In order to use `SO_BINDTODEVICE`, the application must have `CAP_NET_RAW` capabilities.
```
Comment (abhatt-rh) [inline modules/cnf-assigning-a-sriov-network-to-a-vrf.adoc:14]: Applications which use VRFs need to bind to a specific device. The common usage is to use the `SO_BINDTODEVICE` option for a socket. `SO_BINDTODEVICE` binds the socket to a device that is specified in the passed interface name, for example, `eth1`. To use `SO_BINDTODEVICE`, the application must have `CAP_NET_RAW` capabilities.

Comment (abhatt-rh) [reply modules/cnf-assigning-a-sriov-network-to-a-vrf.adoc:14]: A few nits

Comment (aireilly) [reply modules/cnf-assigning-a-sriov-network-to-a-vrf.adoc]: fixed! :)

**Comment by aireilly** [comment]
Ready for QE review

**Review by gkopels — COMMENTED** [review]
lgtm

**Pull request merged**